features/netsocket/UDPSocket.cpp

```diff
@@ -33,6 +33,17 @@ nsapi_protocol_t UDPSocket::get_proto()
     return NSAPI_UDP;
 }
 
+int UDPSocket::join_multicast_group(const SocketAddress &address)
```

Comment:
Not necessarily, although I prefer this style since it looks a lot cleaner to a user and mimics what we used to have.